### ISSUE TITLE
[Feature] Clear axis range selection by action in parallel coordinates plot

### ISSUE THREAD

**Issue by rettrich** [apache/echarts#18446]
### What problem does this feature solve?

There is no action available to clear the axis range selection in parallel coordinates plot. The only option is to click on an axis, but it would be helpful to clear the axis range selection by dispatching an action. 
For now, a workaround is 
`Object.values((echart as any)._componentsMap)`
`  .map(x => (x as any).axisModel)`
`  .filter(Boolean)`
`  .forEach(x => {`
`    x.activeIntervals = [];`
`});`
Here we need to access the internal state of the echart. 
There is an event 'axisareaselected' (https://echarts.apache.org/en/api.html#events.axisareaselected), but there is no action documented to trigger it. 

### What does the proposed API look like?

An action could look like this: 
`dispatchAction( { type: 'clearAxisRangeSelection',  axes: ['x', 'y'] } )`
should clear the axis selection range on the axes named 'x' and 'y' (the name as defined in the `option.parallelAxis[i].name` property). 
Additionally, 
`dispatchAction( { type: 'clearAxisRangeSelection'} )`
could be used to clear the axis selection range on all axes.

**Comment by Ovilia** [comment]
Yes, I think it necessary to provide an action called `'selectAxisRange'`. Are you interested in making a pull request? It may be similar to [`visualMap.selectDataRange`](https://echarts.apache.org/en/api.html#action.visualMap.selectDataRange).

**Comment by CindyCUI423** [comment]
Hi, so I am a first-time contributor and I submitted a pull request to try to fix this issue.

**Comment by Ovilia** [comment]
@CindyCUI423 Thanks for your contribution. I will have a look.

**Comment by Deohgu** [comment]
I am able to clear the `parallelAxis` with brush clear in toolbox,  
as shown below:  

```javascript
    toolbox: {
      feature: {
         brush: {
           icon: {
             clear:
               '<svg path>',
           },
          }
        }
      },
    },
    brush: {
      toolbox: ['clear'],
    },
```
 
But I get that visual artefact also mentioned in the open PR: https://github.com/apache/echarts/pull/19232
Where, the selection is cleared, but one of the axis still has the blue rectangular brush.

![](https://user-images.githubusercontent.com/128704455/277144872-cd05e7ed-8c32-4dca-a849-deb5a32eaf21.gif)

 
My hope was that https://github.com/apache/echarts/pull/19232 would give me the manual tools necessary for me to clear that,  
but from the MR description it seems like that bug is still present?

Any thoughts?

**Comment by Deohgu** [comment]
I found a hacky fix for the above visual bug, which might also shed light on the cause.  

With the above implementation, I now also listen to the brush clear event,  
and call `setOption`. This seems to be enough to remove that selected area brush.

```typescript
    this.chartInstance.on('brush', params => {
      if (params['command'] === 'clear') {
        const option = this.chartInstance.getOption();

        this.chartInstance.setOption(option);
      }
    });
```

**Comment by jamiemac87** [comment]
Adding to @Deohgu suggested workaround if you want to clear all selected ranges/intervals and clear the visual bug without utilising the toolbox brush feature then you can dispatch the following action:
```typescript
    this.chart.dispatchAction({
      type: 'axisAreaSelect',
      intervals: [],
    });
```

However as @Deohgu suggests you still need call `setOption` to remove the visual bug like so:
```typescript
this.chart.on('axisareaselected', (params) => {
  if (!params.intervals.length) {
    this.chart.setOption(this.chart.getOption());
    return;
  }
  ...
});
```
